### ISSUE TITLE
Grab-bag of accessibility issues

### ISSUE THREAD

**Issue by nolanlawson** [missive/emoji-mart#294]
After merging #281 #282 #283 #284, there are still some remaining a11y issues I'd like to tackle:

- [x] title emoji is decorative, should be aria-hidden
- [x] title text should be a title for the whole thing, using `aria-labelledby` or something
- [x] skintone picker should better communicate pressed and dropdown states
- [x] the list of emoji in each category should use `<ul>`/`<li>`
- [x] overall structure of regions/landmarks could be improved

**Comment by nolanlawson** [comment]
There could be additional improvements, especially in the skintone picker, but for now I think these are largely fixed.
